### PR TITLE
Gir files update

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gio/src/auto/file.rs [hunk 1]
```diff
@@ -675,7 +675,7 @@ pub trait FileExt: 'static {
 
     #[doc(alias = "g_file_resolve_relative_path")]
     #[must_use]
-    fn resolve_relative_path(&self, relative_path: impl AsRef<std::path::Path>) -> Option<File>;
+    fn resolve_relative_path(&self, relative_path: impl AsRef<std::path::Path>) -> File;
```
Comment (sdroege) [inline gio/src/auto/file.rs:678]: Should also make a bugfix release with this. Previous bugfix release changed this to an `Option`, which was apparently wrong.

File: gio/src/auto/enums.rs [hunk 2]
```diff
@@ -213,6 +213,8 @@ pub enum CredentialsType {
     NetbsdUnpcbid,
     #[doc(alias = "G_CREDENTIALS_TYPE_APPLE_XUCRED")]
     AppleXucred,
+    #[doc(alias = "G_CREDENTIALS_TYPE_WIN32_PID")]
```
Comment (bilelmoussaoui) [inline gio/src/auto/enums.rs:216]: it needs a version guard

File: glib/gobject-sys/src/lib.rs [hunk 3]
```diff
@@ -1869,6 +1895,55 @@ extern "C" {
     pub fn g_binding_get_target_property(binding: *mut GBinding) -> *const c_char;
     pub fn g_binding_unbind(binding: *mut GBinding);
 
+    //=========================================================================
```
Comment (bilelmoussaoui) [inline glib/gobject-sys/src/lib.rs:1898]: Any reason for not generation those new APIs?

Comment (sdroege) [reply glib/gobject-sys/src/lib.rs:1898]: It needs manual bindings, just like for single signal/property bindings.

File: gio/src/auto/enums.rs (hunk 2 above)
Comment (sdroege) [reply gio/src/auto/enums.rs:216]: Indeed, it's getting boring. In the next days then :)

File: glib/gobject-sys/src/lib.rs (hunk 3 above)
Comment (bilelmoussaoui) [reply glib/gobject-sys/src/lib.rs:1898]: Unfortunate i guess, something for later

Comment (sdroege) [reply glib/gobject-sys/src/lib.rs:1898]: It's quite low level API :)

File: gio/src/auto/enums.rs (hunk 2 above)
Comment (sdroege) [reply gio/src/auto/enums.rs:218]: Done. Anything else @bilelmoussaoui ?

**Pull request merged**